src/packages/README.md

```diff
@@ -26,33 +90,76 @@ the [CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports
 
 **CheriABI packages** are compiled using pure-capability CHERI C/C++, and
 employ fine-grained C/C++ memory protection.
-There are currently over 8,000 CheriABI packages available.
-These packages are considered *experimental*.
 
-These packages are installed in the standard `/usr/local` hierarchy.
+These packages are considered *appropriate for experimental use*.
+Their primary function is to provide necessary dependencies for efforts to port
+software to CheriABI and to support CHERI demonstrations.
+They are suitable for research and development of software to provide spatial
+and temporal memory safety as well as software compartmentalisation.
+They can be also used to investigate potential memory safety issues in
+third-party software that are easier to detect and debug using a CHERI-enabled
+hardware-software stack.
+
+There are currently over 8,000 CheriABI packages available, including:
+* Development utilities: `git`;
+* Editors: `nano`;
+* Libraries: `libnghttp2`, `libressl`, `libssh2`, `lzlib`, `wolfssl`;
+* Networking software: `rsync`;
+* Shells: `dash`.
+
+The packages are installed in the standard `/usr/local` hierarchy as they match
+the base system ABI.
 
 *Note:* These packages compile, but many have CHERI-related warnings that
-have not been audited and only a limited set have been tested.  Bugs
-related to CHERI support can be reported in the
-[CheriBSD ports issue tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+have not been audited and only a limited set have been tested. Bugs related to
+CHERI support can be reported in the [CheriBSD ports issue
+tracker](https://github.com/CTSRD-CHERI/cheribsd-ports/issues).
+While the CheriABI packages are more interesting as they use CHERI memory-safety
+features, you must remember that they might break in run-time and hence might
+not be suitable for critical operations.
+In such case, a corresponding hybrid ABI package might be considered instead of
+a CheriABI package.
 
-## Managing packages in CheriBSD
+## Duplicate packages
 
-*Note:* As of this writing we only provide packages for Morello systems.
-We aim to add RISC-V package sets in the near future.
+Hybrid ABI and CheriABI package repositories include many duplicate packages,
+e.g. `git`.
+If you decided to use a CheriABI package and found out that it crashes, you
+might consider using a hybrid ABI package instead.
 
-Hybrid and CheriABI packages can be installed via two independent instances of
-the package system in CheriBSD, one for each ABI.
-The intention is that, over time, the CheriABI packages will become more
-mature, and hence the preferred collection for day-to-day use.
+You must remember that CheriABI packages have a higher priority than the hybrid
```

Comment:
Probably it’s better to avoid “You” in this context. Perhaps simply lose “You must remember that”?

Reply:
Sounds good. We have lots of "you" in the text. Do you mean context of this particular sentence or the section? I wonder if we should rephrase the rest.

Reply:
I think what I really meant to comment on more was that we shouldn’t suggest that the user has forgotten (i.e., they must remember), but instead simply that the "CheriABI packages have ..”?

Reply:
Yes, that sounds good.